diff --git a/node_modules/nuxt-component-meta/dist/module.mjs b/node_modules/nuxt-component-meta/dist/module.mjs
index 7f51e0d..a0f16a2 100644
--- a/node_modules/nuxt-component-meta/dist/module.mjs
+++ b/node_modules/nuxt-component-meta/dist/module.mjs
@@ -446,7 +446,7 @@ function useComponentMetaParser({
       if (isExcluded(component2)) {
         continue;
       }
-      if (!component2.filePath || !component2.pascalName) {
+      if (!component2.filePath || !component2.pascalName || !component2.filePath.endsWith('.vue')) {
         continue;
       }
       const filePath = resolvePathSync(component2.filePath);
